### ISSUE TITLE
CMVN为什么使用平方和

### ISSUE THREAD

**Issue by Ryuk17** [wenet-e2e/wekws#70]
我看CMVN计算方差使用的是平方和而不是方差，请问这有什么特殊的含义吗，是使用平方和效果会更好一点吗？
` torch.sum(torch.square(mat), axis=0)`

**Comment by blessyyyu** [comment]
Here just store the sum and squre value. Please check the file :  kws/utils/cmvn.py.

**Comment by blessyyyu** [comment]
> 我看CMVN计算方差使用的是平方和而不是方差，请问这有什么特殊的含义吗，是使用平方和效果会更好一点吗？ ` torch.sum(torch.square(mat), axis=0)`

Here just store the sum and squre value. Please check the file : kws/utils/cmvn.py.

**Comment by Ryuk17** [comment]
看到了 谢谢！
